jupyter-kernel-adapter/pom.xml

```diff
@@ -12,12 +12,37 @@
 
 	<dependencies>
 		<dependency>
-			<groupId>org.hamcrest</groupId>
-			<artifactId>hamcrest-all</artifactId>
-			<version>1.3</version>
-			<scope>test</scope>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
+			<version>1.4.6.RELEASE</version>
```

Comment:
These version numbers will be handled by the springframework when the parent artifact is changed.